src/apps/gabac-app/main.cc

```diff
@@ -26,20 +24,13 @@ int main(int argc, char* argv[]) {
                            programOptions.dependencyFilePath);
         } else if (programOptions.task == "decode") {
             gabacify::code(programOptions.inputFilePath,
-                           // RESTRUCT_DISABLE programOptions.configurationFilePath,
                            programOptions.outputFilePath,
                            programOptions.blocksize,
                            programOptions.descID,
                            programOptions.subseqID,
                            true,
                            programOptions.dependencyFilePath);
-        } /* RESTRUCT-DISABLE
-          else if (programOptions.task == "analyze") {
-
-            gabacify::analyze(programOptions.inputFilePath, programOptions.outputFilePath, programOptions.blocksize,
-                              programOptions.maxVal, programOptions.wordSize);
-
-        } */ else {
+        } else {
             GABAC_DIE("Invalid task: " + std::string(programOptions.task));
```

Comment:
GABAC_DIE has been removed on the develop branch and has been replaced by UTILS_DIE